cdap-ui/app/features/etlapps/controllers/create-ctrl.js

```diff
@@ -1,8 +1,11 @@
 angular.module(PKG.name + '.feature.etlapps')
-  .controller('ETLAppsCreateController', function($scope, MyDataSource, $alert, $bootstrapModal, $state, ETLAppsApiFactory) {
+  .controller('ETLAppsCreateController', function($scope, MyDataSource, $alert, $bootstrapModal, $state, ETLAppsApiFactory, $filter) {
     var apiFactory = new ETLAppsApiFactory($scope);
 
     $scope.ETLMetadataTabOpen = true;
+    $scope.ETLSourcesTabOpen = true;
+    $scope.ETLTransformsTabOpen = true;
+    $scope.ETLSinksTabOpen = true;
```

Comment:
We need to be able to close them. Its just the first time we need to keep it open.